### PR TITLE
Feature/wasm transfer tokens

### PR DESCRIPTION
**Pull request by OBorce**
- new functions to transfer/burn tokens

### CODE REVIEW

File: wasm-wrappers/src/lib.rs [hunk 1]
```diff
@@ -355,6 +355,24 @@ pub fn encode_output_transfer(
     Ok(output.encode())
 }
 
+/// Given a destination address, an amount, token ID and a network type (mainnet, testnet, etc), this function
+/// creates an output of type Transfer for tokens, and returns it as bytes.
+#[wasm_bindgen]
+pub fn encode_output_token_transfer(
```
Comment (TheQuantumPhysicist) [inline wasm-wrappers/src/lib.rs:361]: This doesn't seem to be tested... please make sure everything has tests.

File: wasm-wrappers/src/lib.rs [hunk 2]
```diff
@@ -355,6 +355,24 @@ pub fn encode_output_transfer(
     Ok(output.encode())
 }
 
+/// Given a destination address, an amount, token ID and a network type (mainnet, testnet, etc), this function
```
Comment (TheQuantumPhysicist) [inline wasm-wrappers/src/lib.rs:358]: Two things:

1. I just noticed that we can specify (in a more clearer manner) that amounts are in decimal. This applies to all functions. So just add after `an amoun` in parenthesis (in decimal as a string).
2. Please mentioned after token ID in parenthesis (in address form)

Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:358]: Same applies to all functions

Comment (OBorce) [reply wasm-wrappers/src/lib.rs:358]: amounts are a separate struct Amount which is constructed from atoms.

**Review by TheQuantumPhysicist — APPROVED** [review]
Please add positive tests too

**Pull request merged**